paddle/framework/net_design.md

```diff
@@ -0,0 +1,159 @@
+# Network Design
+
+`Network` is the container and controller of a set of operators in a network, 
```

Comment:
`Network` is the container and controller of a set of operators in a network. 
=>
`Network` is the container and controller of a set of operators.